src/components/yupSchemas/RegistrationSchema.ts

```diff
@@ -0,0 +1,14 @@
+import * as yup from 'yup';
+
+export const registrationSchema = yup.object().shape({
+  email: yup.string().email('Email musi mieć odpowiedni format').required('Email jest wymagany'),
+  password: yup.string().required('Hasło jest wymagane').min(8, 'Hasło musi zawierać minimum 8 znaków'),
```

Comment:
dodałbym jeszcze max ilość znaków przy haśle - do przegadania ile chcemy lub zaproponuj coś - zrobimy wtedy tak samo na backendzie

Reply:
Dałbym 100 bądź 255, w zależności do maksymalnej ilości znaków w bazie danych

Reply:
To tak nie działa. Na backendzie korzystamy z  https://www.npmjs.com/package/bcrypt do hashowania hasła.
Bcrypt ma maksymalnie 72 bajty które może zahashować - wszystko powyżej utnie i może być problem :-)
Jeśli założymy że korzystamy z liter alfabetu łacińskiego, cyfer i podstawowych symboli to o ile dobrze przeczytałem, to każdy z tych znaków reprezentuje 1 bajt, wiec maksymalna długość hasła to 72 znaki.

Proponuję może zrobić od 10 do 24 znaków + to co wyżej (litery lfabetu łacińskiego, cyfry, podstawowe znaki) + jak masz pomysł to możesz wymusić min jedna duża literę, jedna cyfrę, jeden znak. :-)

Reply:
Wytyczne:
Co najmniej 1 małą literę (a-z)
Co najmniej 1 dużą literę (A-Z)
Co najmniej 1 cyfrę (0-9)
Co najmniej 1 znak specjalny, taki jak: !,@,#,$,%,^,&,*
Minimalna ilość znaków 8
Maksymalna ilość znaków 24

Tak przygotowuje backend :)